modules/drivers/robosense/README.md

```diff
@@ -0,0 +1,202 @@
+# **Robosense LiDAR Driver**
```

Comment:
It would be better if READEME.md could also be provided in English, then the Chinese version can be renamed to `README_cn.md`.